napari/utils/transforms/transform_utils.py

```diff
@@ -1,10 +1,13 @@
+from typing import Tuple
+
 import numpy as np
+import numpy.typing as npt
 import scipy.linalg
 
 from napari.utils.translations import trans
 
 
-def compose_linear_matrix(rotate, scale, shear) -> np.array:
+def compose_linear_matrix(rotate, scale, shear) -> npt.NDArray:
```

Comment:
Why `npt.NDArray` when it is always `np.ndarray`?

Reply:
This is a generic version of `np.ndarray` (see https://numpy.org/doc/1.21/reference/typing.html#numpy.typing.NDArray), which makes it easier to add data types as a generic type hint later down the line. (also to be consistent with the rest of the typing where I've been asked in other PRs to use `numpy.typing` 😄 )